datahost-ld-openapi/test/tpximpact/datahost/ldapi/models/series_test.clj

```diff
@@ -24,26 +24,31 @@
             (is (= status 404))
             (is (= body "Not found"))))))
 
-    (let [incoming-jsonld-doc {"@context"
-                               ["https://publishmydata.com/def/datahost/context"
-                                {"@base" "https://example.org/data/"}],
-                               "dcterms:title" "A title"}
-          augmented-jsonld-doc (sut/normalise-series {:series-slug "new-series"}
-                                                     incoming-jsonld-doc)]
+    (let [request-ednld {"@context"
+                         ["https://publishmydata.com/def/datahost/context"
+                          {"@base" "https://example.org/data/"}],
+                         "dcterms:title" "A title"}
+          normalised-ednld {"@context"
+                            ["https://publishmydata.com/def/datahost/context"
+                             {"@base" "https://example.org/data/"}],
+                            "@type" "dh:DatasetSeries"
+                            "@id" "new-series"
+                            "dh:baseEntity" "https://example.org/data/new-series/"
+                            "dcterms:title" "A title"}]
       (testing "A series can be created and retrieved via the API"
 
         (let [response (http/put
                         "http://localhost:3400/data/new-series"
                         {:content-type :json
-                         :body (json/write-str incoming-jsonld-doc)})]
+                         :body (json/write-str request-ednld)})]
           (is (= (:status response) 201))
```

Comment:
A minor thing, but improves the developer experience: the expected value goes first - otherwise  the test report can be confusing. So, it should be `(is (= 201 (:status response)))` etc.